crates/nu-std/std/mod.nu

```diff
@@ -335,3 +335,16 @@ export def repeat [
 
     1..$n | each { $item }
 }
+
+# return a null device file.
+#
+# # Examples
+#     run a command and ignore it's stderr output
+#     > cat xxx.txt e> (null-device)
+export def null-device []: any -> string {
```

Comment:
should it take `any`?

Reply:
Actually I'm not really sure, it doesn't deal with any pipeline input.  Maybe changed to nothing?